packages/plugin-rok/docs/jspsych-rok.md

```diff
@@ -0,0 +1,230 @@
+# jspsych-rok plugin
+
+This plugin displays oriented objects (oobs) that move and allows the subject to report the primary direction of motion or the primary orientation by pressing a key on the keyboard. 
+The stimulus can be displayed until a keyboard response is given or until a certain duration of time has passed. Oobs are objects that have an orientation (e.g. triangles).
+
+We would appreciate it if you cited this paper when you use the plugin: 
+
+<b>Strittmatter, Y., Spitzer, M., & Kiesel, A. (2021, July 12). A Random-Object-Kinematogram Plugin for Web-Based Research: Implementing Oriented Objects Enables Varying Coherence Levels and Stimulus Congruency Levels. DOI: [10.31234/osf.io/hmq4u]</b>
```

Comment:
Can you use markdown instead of html for font styles?

Wrapping in ** will work